src/signals/incident/components/form/MapSelectors/Caterpillar/CaterpillarLayer/CaterpillarLayer.tsx

```diff
@@ -25,8 +30,8 @@ export const CaterpillarLayer: FC = () => {
       const coordinates = featureToCoordinates(feature.geometry as Geometrie)
       // Caterpillar layer renders only a single feature type (oak tree)
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      const featureType = meta.featureTypes.find(
-        ({ label }) => label === 'Eikenboom'
+      const featureType = meta.featureTypes!.find(
```

Comment:
Maybe change the `Meta` type for this use case instead of setting a non-null assertion?